X/XyceWrapper/build_tarballs.jl

```diff
@@ -40,8 +47,8 @@ products = [
 dependencies = [
     Dependency("Xyce_jll"; compat="^7.4.0"),
     Dependency("libcxxwrap_julia_jll"),
-    BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version)),
+    BuildDependency("libjulia_jll"),
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"7")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat = "1.6", preferred_gcc_version=v"7")
```

Comment:
I just realised this required Julia v1.6 without changing the version number, which is a violation of second point of #2763.  And this is why https://github.com/JuliaRegistries/General/pull/54942 is stalling.

Reply:
Oh that is easy enough fix :). Thanks!